docs/cristin-proxy-swagger.yaml

```diff
@@ -88,6 +88,99 @@ paths:
             type: number
             default: '5'
           example: '10'
+        - name: institution
```

Comment:
In code, you are using `public static final String CRISTIN_INSTITUTION_ID = "cristin_institution_id";` as input param, but when adding to query you are using `private static final String CRISTIN_QUERY_PARAMETER_INSTITUTION = "institution";`

So either you should change input param to `institution` or update swagger to use `cristin_institution_id`

Reply:
`institution` is the param name in Cristin

Reply:
I will use institution for my purposes.